src/common/game_def.hpp

```diff
@@ -9,13 +9,15 @@ struct keys {
     sf::Keyboard::Key up, down, left, right;
 };
 
-struct position {
+struct network_game_object {
     sf::Uint32 id;
     float x_pos;
     float y_pos;
+    float size;
```

Comment:
things are getting mixed up. size is talking about the radius of a group, but the id is just a client id right now right?

Reply:
groups != clients is my thought.

Reply:
I wonder if we need to start designing for grouping/ungrouping. like maybe this should have a field added for "member clients" or member_players. I can imagine many kind of network_game_objects, such as mines or obstacles. Since we know these are part of the game we can probably save ourselves refactoring later, if we try to create something more general now. (usually Im against designing too generally too early, but since we know the game design, I think it will probably save us work overall)

Reply:
Right now, per client we have 1 group and 1 player, so I'm using client_id, group_id and player_id interchangeably. 
Of course this isn't going to work in the long term but I think this is a reasonable step forward for this change.

The code change that supports communicating these ids distinctly is non trivial.

Reply:
fair enough, I'd put that in a comment `I'm using client_id, group_id and player_id interchangeably.`
and explain why, it's a little confusing